projects/1.16/assets/minecraft-earth-mod/minecraft_earth_mod/lang/zh_cn.json

```
@@ -1 +1,190 @@
+  "entity.minecraft_earth_mod.furnace_golem": "熔炉傀儡",
+  "entity.minecraft_earth_mod.fancy_chicken": "华丽鸡",
+  "entity.minecraft_earth_mod.dyed_cat": "染色猫",
+  "entity.minecraft_earth_mod.dried_muddy_pig": "干燥的泥猪",
+  "entity.minecraft_earth_mod.cookie_cow": "曲奇牛",
+  "entity.minecraft_earth_mod.cluckshroom": "咯菇",
+  "entity.minecraft_earth_mod.brule_cow": "奶油牛",
+  "entity.minecraft_earth_mod.bronzed_chicken": "青铜鸡",
+  "entity.minecraft_earth_mod.boulder_zombie": "攀岩僵尸",
+  "entity.minecraft_earth_mod.bone_spider": "骨蜘蛛",
+  "entity.minecraft_earth_mod.ashen_cow": "灰白牛",
+  "entity.minecraft_earth_mod.amber_chicken": "琥珀鸡",
+  "entity.minecraft_earth_mod.albino_cow": "白化牛",
+  "entity.minecraft_earth_mobs.muddy_pig": "泥猪",
+  "block.minecraft_earth_mod.stone_tappable": "石头可点击物",
+  "block.minecraft_earth_mod.spruce_tappable": "云杉木可点击物",
+  "block.minecraft_earth_mod.ruby_shop_block": "红宝石交易方块",
+  "block.minecraft_earth_mod.ruby_ore": "红宝石矿石",
+  "block.minecraft_earth_mod.ruby_block": "红宝石块",
+  "block.minecraft_earth_mod.rainbow_wool": "彩虹羊毛",
+  "block.minecraft_earth_mod.rainbow_carpet": "彩虹地毯",
+  "block.minecraft_earth_mod.pink_daisy": "粉红色雏菊",
+  "block.minecraft_earth_mod.oak_tappable": "橡木可点击物",
+  "block.minecraft_earth_mod.mud_converter": "泥浆转化器",
+  "block.minecraft_earth_mod.mud": "泥浆",
+  "block.minecraft_earth_mod.grass_tappable": "草可点击物",
+  "block.minecraft_earth_mod.furnace_golem_light": "熔炉傀儡的光源",
+  "block.minecraft_earth_mod.chest_tappable": "箱子可点击物",
+  "block.minecraft_earth_mod.carved_melon": "雕刻过的西瓜",
+  "block.minecraft_earth_mod.buttercup": "金凤花",
+  "block.minecraft_earth_mod.boulder_zombie_climbing_block": "攀岩僵尸正爬的方块",
+  "block.minecraft_earth_mod.birch_tappable": "白桦木可点击物",
+  "biome.minecraft_earth_mod.plains_build_plate": "平原（建造模板）",
+  "biome.minecraft_earth_mod.mountains_build_plate": "山脉（建造模板）",
+  "biome.minecraft_earth_mod.jungle_build_plate": "丛林（建造模板）",
+  "biome.minecraft_earth_mod.icy_build_plate": "冰原（建造模板）",
+  "biome.minecraft_earth_mod.forest_build_plate": "森林（建造模板）",
+  "advancements.rubies.title": "RWBY",
+  "advancements.rubies.descr": "获得一颗红宝石！",
+  "advancements.place_ruby_block.title": "♫Red Like Roses♪",
+  "advancements.place_ruby_block.descr": "放置一块红宝石块！",
+  "advancements.p_2_p.title": "P 2 P",
+  "advancements.p_2_p.descr": "至少你没在这挥洒真钱",
+  "advancements.minecraft_earth_advancement.title": "我的世界：地球",
+  "advancements.minecraft_earth_advancement.descr": "感谢使用我的模组！ - Derec_",
+  "advancements.hogsmeade.title": "霍格莫德",
+  "advancements.hogsmeade.descr": "“真想给你带些黄油啤酒，真正能让你暖和起来。”",
+  "advancements.get_rainbowwool.title": "粉粉蓬蓬独角兽",
+  "advancements.get_rainbowwool.descr": "♫在彩虹上跳舞♪",
+  "advancements.get_mud.title": "在泥坑里跳来跳去！",
+  "advancements.get_mud.descr": "经典之作",
+  "advancements.get_horn.title": "角了",
+  "advancements.get_horn.descr": "（不要吹响号角）",
+  "advancements.get_dimension_key.title": "游戏终末之钥",
+  "advancements.get_dimension_key.descr": "远古的光与暗之魂已经被释放了（沉声）……我去，搞错游戏了",
```

```suggestion
  "advancements.get_dimension_key.descr": "古老的光明与黑暗之魂已经释放……我去，搞错游戏了",
```
https://terraria.wiki.gg/zh/wiki/%E8%A1%80%E8%82%89%E5%A2%99
别乱给译文加戏